content/manuals/ai/gordon/_index.md

```diff
@@ -194,6 +197,10 @@ across several dimensions:
 - Portability
 - Resource efficiency
 
+### Migrate a Dockerfile to DHI
+
+See [Use Gordon AI](/manuals/dhi/how-to/migrate.md#use-gordon-ai) in the DHI section.
```

Comment:
Can we add a brief intro to this section to provide context for users? Suggest we use the includes file to also list the instructions here on how to migrate a dockerfile to a DHI.